lib/dicom_net/association.ex

```diff
@@ -89,17 +98,31 @@ defmodule DicomNet.Association do
     Logger.info("Accepting association")
     association_data = accept_associate_request(associate_request)
 
-    new_state =
-      state
-      |> Map.put(:state, :association_established)
-      |> Map.put(:association, association_data)
+    case association_data.calling_ae_title do
+      "FINDSCU" ->
```

Comment:
I assume this is coming from your other experiments and is not supposed to be in here?